assets/src/js/frontend/give-stripe-sepa.js

```diff
@@ -30,6 +24,22 @@ document.addEventListener( 'DOMContentLoaded', function( e ) {
 	Array.prototype.forEach.call( formWraps, function( formWrap ) {
 		const formElement = formWrap.querySelector( '.give-form' );
 
+		// Bailout, if `form_element` is null.
```

Comment:
Same here, please explain this.

Reply:
Added detailed comments as it is expected scenario